### DISCUSSION TITLE
lego certificate renewal fails - how to revoke letsencrypt certificate with lego and start over

### DISCUSSION THREAD

**Discussion by tacoma50** [go-acme/lego#2380] [Q&A]
I am trying to renew a cert against the STAGING server like this:
```
/opt/bitnami/letsencrypt/lego --tls --email="joe@xyz.com" --server=https://acme-staging-v02.api.letsencrypt.org/directory --domains="xyz.com" --path="/opt/bitnami/letsencrypt" renew --days 90 
```
I am getting this error: 
```
While validating order as a replacement an error occurred :: cannot indicate an order replaces certificate with serial "2b54653073af4b3d54fcb0b4fdf0c8d4ba6c", which already has a replacement order.
```
A this point I just want to start over with a whole new cert. How do I revoke the current cert using lego and start over by requesting a new cert.

**Comment by ldez** [comment]
Hello,

you can use the revoke command: https://go-acme.github.io/lego/usage/cli/options/index.html

**Comment by tacoma50** [comment]
OK I tried revoke as follows, but what if I don't have the current valid crt file?

/opt/bitnami/letsencrypt/lego  --tls --email="joe@xyz.com" --domains="xyz.com"  --server=https://acme-staging-v02.api.letsencrypt.org/directory --path="/opt/bitnami/letsencrypt"  revoke --reason 0
2024/12/11 14:47:59 Trying to revoke certificate for domain xyz.com
2024/12/11 14:47:59 Error while revoking the certificate for domain xyz.com
	open /opt/bitnami/letsencrypt/certificates/xyz.com.crt: no such file or directory

**Reply by ldez** [reply]
This means you don't have the certificate locally, so you cannot revoke it.
So you should try to create a certificate.

**Comment by tacoma50** [comment]
OK Thanks.
Just curious, is there a way to revoke cert if you no longer have it, but you still have your account for the cert?
